aws/policy/security-services.yaml

```diff
@@ -12,6 +12,7 @@ Statement:
       - iam:DeleteInstanceProfile
       - iam:AddRoleToInstanceProfile
       - iam:RemoveRoleFromInstanceProfile
+      - iam:DisassociateIamInstanceProfile
```

Comment:
I don't see this one in the [docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_identityandaccessmanagement.html), only for `ec2:DisassociateIamInstanceProfile`.